lib/bob/job/otp_checker.ex

```diff
@@ -1,28 +1,31 @@
 defmodule Bob.Job.OTPChecker do
   @repo "erlang/otp"
   @linuxes ["ubuntu-18.04", "ubuntu-20.04", "ubuntu-22.04"]
+  @arches ["amd64", "arm64"]
 
   def run(_type) do
     for linux <- @linuxes,
-        {ref_name, ref} <- Bob.GitHub.diff(@repo, "builds/otp/#{linux}"),
-        build_ref?(linux, ref_name),
-        do: Bob.Queue.add(Bob.Job.BuildOTP, [ref_name, ref, linux])
+        arch <- @arches,
+        {ref_name, ref} <- Bob.GitHub.diff(@repo, "builds/otp/#{linux}-#{arch}"),
```

Comment:
Let's use this paths to make it easier to do path rewrites on the CDN.

```suggestion
        {ref_name, ref} <- Bob.GitHub.diff(@repo, "builds/otp/#{arch}/#{linux}"),
```